.github/workflows/compile_test.yml

```diff
@@ -0,0 +1,43 @@
+name: Compile and Test
+
+on: [push]
+
+env:
+  CI: true # disables SBT super shell, which has problems with CI environments
+
+jobs:
+  build:
+    name: Scala ${{ matrix.scala }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        scala: [2.12.10, 2.13.1]
+
+    env:
+      SCALA_VERSION: ${{ matrix.scala }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
```

Comment:
For consistency with our other repos, do you mind changing this actions to `olafurpg/setup-scala@v7` ?

Reply:
Changed it in e6fd242.
Generally, if I don't have a specific requirement that could only be satisfied by using another action, I prefer to use official ones. But as we are using this elsewhere, I reckon it is OK.